From 65221a41f27640f37500a90a8e96011c03836830 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 20 Apr 2026 15:26:36 +0300
Subject: [PATCH] Update for rpi4 on CompuLab boards

Upstream-Status: Pending

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/raspberrypi/rpi/rpi.env | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/board/raspberrypi/rpi/rpi.env b/board/raspberrypi/rpi/rpi.env
index 9ac9d6768ca..d045002e6c6 100644
--- a/board/raspberrypi/rpi/rpi.env
+++ b/board/raspberrypi/rpi/rpi.env
@@ -76,5 +76,9 @@ scriptaddr=0x05400000
 pxefile_addr_r=0x05500000
 fdt_addr_r=0x05600000
 ramdisk_addr_r=0x05700000
+autoload=no
 
 boot_targets=mmc usb pxe dhcp
+vid=0x0a5c
+pid=0x0104
+run_msd=ums 0 mmc 0
-- 
2.34.1

